docs/DISTRIBUTION.md

```diff
@@ -1 +1,92 @@
 # CWA-Server Distribution Service
+
+## Configuration Properties
+
+All of the configurable parameters, that are used throughout this service can be found in the [`application.yaml`](/services/distribution/src/main/resources/application.yaml) file.
+This configuration file is divided into different sub-categories:
+
+- Some general configuration, which isn't divided into sub-categories
+- `paths` - Local Paths, that are used during the export creation
+- `tek-export` - Configuration for the exported archive, that is saved on the S3-compatible storage
+- `api` - API Configuration, configures the API, which is used by the mobile app to query diagnosis keys
+- `signature` - Signature Configuration, used for signing the exports
+- `objectstore` - Configuration for the S3 compatible object storage
+
+## Object Store
+
+The communication with the S3 compatible object storage, that is hosted by Deutsche Telekom is done through the AWS SDK
+v2. The files, that will be updated to the S3 compatible storage are created on local storage first (output path defined
+in application configuration) and are then uploaded to the storage provider.
+
+### Headers
+
+Up to three headers are set during upload, depending on the configuration properties.
+
+#### `Cache-Control`
+
+Defines the maximum amount of time a published resource is considered fresh, when held in cache. Will be set for each
+upload, default is 300 seconds.
+
+#### `x-amz-acl`
+
+Defines the canned ACL for the uploaded file. Is only set if [`set-public-read-acl-on-put-object`] is set to true in the
+configuration properties. Will be `public-read` in that case, which grants full control to the Owners and read-access to
+AllUsers.
+
+#### `cwa-hash`
+
+Custom header, which adds the SHA-256 hash of the archive contents. This header is used to determine, whether a file
+should be uploaded or not. If the hash for the file is the same as the hash available on the S3 compatible storage it
+will not be uploaded, since the contents of that specific file did not change, so there is no need to re-upload the
+file. If the hash differs, or the file is not available on the S3 compatible storage the file will be uploaded.
+
+This header is needed, since it is not possible to create byte-identical archives, due to the signing algorithms.
```

Comment:
```suggestion
This header is needed, since it is not possible to create byte-identical archives when using ECDSA, since it is non-deterministic.
```